scripts/gen_all_resource_docs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
I used this script to generate _all_ of these docs, so I figured I would add this to the repo as well. I don't anticipate this being in service for too long, though. Once we switch to automatically generating the docs from upstream when a provider release is made, this would not be needed.